src/main/java/com/github/fishio/PowerUpSpawner.java

```diff
@@ -85,7 +79,7 @@ public PowerUp getRandomPowerUp() {
 		
 		//Creating the BoundingBox for the PowerUp with a on the
 		//top of the screen and with a random x location.
-		int x = rand.nextInt(maxX - minX + 1) + minX;
+		int x = rand.nextInt((int) settings.getDouble("SCREEN_WIDTH") - 1);
```

Comment:
The screen width is the width including the border on the sizes, therefore I subtracted 1 to account for that. But 2 might have been a better value.

<logic